relay-filter/Cargo.toml

```diff
@@ -17,6 +17,7 @@ relay-common = { path = "../relay-common" }
 relay-event-schema = { path = "../relay-event-schema" }
 relay-protocol = { path = "../relay-protocol" }
 relay-ua = { path = "../relay-ua" }
+relay-sampling = { path = "../relay-sampling" }
```

Comment:
This is unfortunate. Could you try if anything breaks when we move `condition.rs` to `relay-protocol` instead?

Reply:
Should we make the move in the same PR?

Reply:
Can also be a separate one. Would definitely make the diff more manageable.